Example/Controllers/SwiftUI/Features/Settings/Features/MiniAppSettingsContactsView.swift

```diff
@@ -65,6 +67,20 @@ struct MiniAppSettingsContactsView: View {
     func index(for contact: MAContact) -> Int? {
         return contacts.firstIndex(where: { $0.id == contact.id })
     }
+
+    func removeEmptyEmailStrings(for contact: MAContact) -> MAContact {
+        var newContact = contact
+        if newContact.allEmailList != nil {
+            if newContact.allEmailList?.first ?? "" == "" {
+                newContact.allEmailList = nil
+            } else {
+                newContact.allEmailList = newContact.allEmailList?.filter { $0 != "" }
+            }
+            return newContact
+        } else {
+            return newContact
+        }
```

Comment:
Sorry I did not notice this before, this should work too
```suggestion
        if newContact.allEmailList?.first ?? "" == "" {
            newContact.allEmailList = nil
        } else {
            newContact.allEmailList = newContact.allEmailList?.filter { $0 != "" }
        }
        return newContact
```

Reply:
Thanks, I did find a flaw in this and fixed it in new commit. ;)